### PR TITLE
Adding expression pattern widgets for antibody, transgene and construct

### PR DESCRIPTION
**Pull request by adamjohnwright**
This is simply the table that was made for life stage page expression widget. There are a couple features to make it slightly more complete and has added expression widgets for antibody, transgene and construct

### CODE REVIEW

**Review by mgrbyte — APPROVED** [review]
indent is wonky in anon functions e.g line 17), but other than that :+1:

File: src/rest_api/classes/expr_pattern/core.clj [hunk 1]
```diff
@@ -6,14 +6,25 @@
...
+                    (or
+                      (some->> (:expr-pattern/subcellular-localization ep)
+                               (sort)
+                               (str/join "<br />")
+                               (not-empty))
+                      (some->> (:expr-pattern/remark ep)
+                               (map :expr-pattern.remark/text)
+                               (sort)
+                               (str/join "<br />")
+                               (not-empty))))}))
```
Comment (sibyl229) [inline src/rest_api/classes/expr_pattern/core.clj]: Not sure if `(not-empty)` works on a string. It takes a collection as an input. But if you think it's fine, then it's okay.

File: src/rest_api/classes/transgene/widgets/expression.clj [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+(defn marker-for [t]
+  {:data (some->> (:transgene/marker-for t)
+                  (first)
+                  (:transgene.marker-for/text))
+   :description "string decribing what the transgene is a marker for"})
+
+(defn expression-patterns [t]
+  {:data (some->> (:expr-pattern/_transgene t)
+                  (map (fn [ep]
```
Comment (sibyl229) [inline src/rest_api/classes/transgene/widgets/expression.clj:17]: `map` returns an array, possibly empty one, and you probably need to use `seq` to convert it to `nil`

Comment (mgrbyte) [reply src/rest_api/classes/transgene/widgets/expression.clj:17]: Is fine because `some->>` will return nil if the last expression is empty or nil.

**Pull request merged**